STORM-IRIT/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ex
+
+./pdpcComputeMultiScaleFeatures -i input/$1 -v -o output/output
+./pdpcSegmentation -i input/$1 -s output/output_scales.txt -f output/output_features.txt -o output/output -v
```

Comment:
It will be better to use input-based names for all outputs (maybe except temporary ones like this) to prevent overriding of the results by second launch